includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -475,4 +501,61 @@ private function formatLocalNavData( array $items, int $nestingLevel ): array {
 	private function getSpecialPageURL( $name ): string {
 		return SpecialPage::getTitleFor( $name )->getLocalURL();
 	}
+
+	private function isFandomStoreCommunity( $wikiId ) {
+		global $wgFandomStoreMap;
+		return array_key_exists( $wikiId, $wgFandomStoreMap );
+	}
+
+	private function getFandomStoreData() {
+		global $wgMemc, $wgCityId;
+	
+		$memcKey = wfMemcKey( DesignSystemApiController::MEMC_PREFIX_FANDOM_STORE, $wgCityId );
+
+		$cachedStoreData = $wgMemc->get( $memcKey );
+
+		// if results are empty, return null
```

Comment:
`return !empty( $cachedStoreData->results ) ? $this->formatFandomStoreData( $storeData->results ) : null;`

Reply:
Slightly less verbose but not quite as readable. I'll go with your approach for now.